lib/annotations/Unique.ts

```diff
@@ -1,12 +1,27 @@
 import 'reflect-metadata';
 import {addAttributeOptions} from "../services/models";
 
+export type uniqueAnnotation = (target: any, propertyName: string) => void;
+
 /**
- * Sets unique option true for annotated property.
+ * Sets unique option for annotated property.
  */
-export function Unique(target: any, propertyName: string): void {
+export function Unique(keyCombinator: string): uniqueAnnotation;
```

Comment:
I think we should provide all possible options here (see: https://github.com/RobinBuschmann/sequelize-typescript/blob/master/lib/interfaces/IPartialDefineAttributeColumnOptions.ts#L18) What do you think?